examples/with-apollo/lib/withData.js

```diff
@@ -48,14 +48,7 @@ export default ComposedComponent => {
         Head.rewind()
 
         // Extract query data from the Apollo store
-        const state = apollo.getInitialState()
-
-        serverState = {
-          apollo: {
-            // Only include the Apollo data state
-            data: state.data
-          }
-        }
+        serverState = apollo.cache.extract()
```

Comment:
I can see that, differently than in #3029, you don't wrap the apollo data with an `apollo.data` key. Is there any advantage on that?

Reply:
@zvictor didn't seem to be needed anymore, but is a very good question.

Reply:
thanks!
@jthegedus would you have an opinion on that?

Reply:
People may want to generate other data in an SSR I left it as is because I wasn't sure what other SSR state people may be generating and appending to a

Reply:
@jthegedus yeah you are right! I'll change that 👍

Reply:
@jthegedus and @zvictor just updated with the `state.apollo.data` format without loosing the SSR.